### PR TITLE
Makes bat hits no longer stunlock

### PR DESCRIPTION
**Pull request by Sparkezel**
## About The Pull Request

Sets `gentle` in `target.throw_at` in the bat code to true.
Increases bat damage by 2 in recompensation.

## Why It's Good For The Game
Wooden bats are 2nd best melee weapon in game, just next to stun batons. They cost 5 wooden planks which makes them ridiculously cheap and available to everyone, and the worst thing about them is how easily they can just stunlock people out of nowhere. I did what I believe was the best way to balance them out

## Changelog
:cl:
balance: Bats no longer stunlock people. Increased bat damage by 2.
/:cl:

### CODE REVIEW

**Comment by InsaneRed** [comment]
we need this anyone who says otherwise is an assistant main

File: code/game/objects/items/weaponry.dm
Comment (Fikou) [inline code/game/objects/items/weaponry.dm:695]: thats not how you do args

Comment (Sparkezel) [reply code/game/objects/items/weaponry.dm:695]: how do I do it better then?

**Comment by rdragan** [comment]
Botany nerf

File: code/game/objects/items/weaponry.dm
Comment (MrDoomBringer) [reply code/game/objects/items/weaponry.dm]: named arguments

**Comment by interestingusernam3** [comment]
>Botany nerf

emagged_beepsky.txt

**Comment by Tumesuo** [comment]
no soul

**Review by coiax — APPROVED** [review]
I'm planning to merge this unless another maintainer objects. Bats are just really strong given you can get one with a wrench and an unlooted library.

**Comment by TemporalOroboros** [comment]
Fun fact! The gentle argument in throwing only really effects getting thrown into other people. If they hit a wall or machine they will still get stunned!

**Comment by Sparkezel** [comment]
> Fun fact! The gentle argument in throwing only really effects getting thrown into other people. If they hit a wall or machine they will still get stunned!

I checked it with monkies and they could both attack me and I could attack them, so it seems to be fine

**Review by Rohesie — APPROVED** [review]

**Review by Timberpoes — APPROVED** [review]

**Pull request merged**
